<x-slot name="content">
                         <!-- Authentication -->
+                        <x-dropdown-link :href="route('admin.account.info')" :active="request()->routeIs('admin.account.info')">
+                                {{ __('My Account') }}
+                        </x-dropdown-link>
                         <form method="POST" action="{{ route('logout') }}">
                             @csrf
@@ -94,6 +97,9 @@
             </div>
<div class="mt-3 space-y-1">
+                <x-responsive-nav-link :href="route('admin.account.info')" :active="request()->routeIs('admin.account.info')">
+                        {{ __('My Account') }}
+                </x-responsive-nav-link>
                 <!-- Authentication -->
                 <form method="POST" action="{{ route('logout') }}">
                     @csrf
